src/utils.ts

```diff
@@ -2020,6 +2019,12 @@ export class newAxis {
     return Math.abs(this.origin.x - this.end.x);
   }
 
+  private get drawingColor(): string {
+    let color = this.strokeStyle;
+    if (this.mouseStyleON) { color = this.isHovered ? this.hoverStyle : this.isClicked ? this.clickedStyle : this.strokeStyle };
```

Comment:
I think I've already seen this somewhere else in another PR. Can it be mutualized ?

Reply:
Axis can be seen as an extent of Shape, which has already this kind of choice. So i see what you mean but i hope you will let me deal with that later, when the current plot_data will be less than 5000 rows